csc_335_restaurant/src/test/ServerTests.java

```diff
@@ -21,15 +21,19 @@ void testName() {
 	}
 	
 	@Test
-	void testNumSeated() {
+	void testToString() {
 		Server s = new Server("Christopher");
-		Assertions.assertEquals(s.getSection(), 0);
+		Assertions.assertEquals(s.toString(), "Server: Christopher has made $0.0 in tips.\n");
 	}
 	
 	@Test
-	void testToString() {
-		Server s = new Server("Christopher");
-		Assertions.assertEquals(s.toString(), "Server: Christopher has: 0 people sat in their section, and has made $0.0 in tips.\n");
+	void testGetTables() {
+		Server s = new Server("Christopher"); 
+		s.addTable(0);
+		s.addTable(1);
+		s.addTable(4);
+		s.removeTable(0);
+		Assertions.assertEquals(s.getTables().size(), 2);
```

Comment:
To ensure the behavior, this test should compare with other array list so that we would know exactly what is within the tables array
```suggestions
@Test
	void testGetAddRemoveTables() {
		Server s = new Server("Christopher"); 
		s.addTable(0);
		s.addTable(1);
		s.addTable(4);
		ArrayList<Integer> expectedTables = new ArrayList<>();
		expectedTables.add(0);
		expectedTables.add(1);
		expectedTables.add(4);
		Assertions.assertEquals(expectedTables, s.getTables());
		s.removeTable(4);
		expectedTables.remove(Integer.valueOf(4));
		Assertions.assertEquals(s.getTables().size(), 2);
		Assertions.assertEquals(expectedTables, s.getTables());
	}
```